content/components/breadcrumbs.mdx

```diff
@@ -16,7 +16,7 @@ import {AccessibilityLink} from '~/src/components/accessibility-link'
 <img
   width="960"
   alt="An image of the breadcrumb component"
-  src="https://user-images.githubusercontent.com/586552/204619264-27de7458-1a77-445f-bc56-83472bf30bfe.png"
+  src="https://private-user-images.githubusercontent.com/7265547/308463005-4bf1f4bd-e067-4e67-bbc4-a2d5119ccf43.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkxMDY1MzUsIm5iZiI6MTcwOTEwNjIzNSwicGF0aCI6Ii83MjY1NTQ3LzMwODQ2MzAwNS00YmYxZjRiZC1lMDY3LTRlNjctYmJjNC1hMmQ1MTE5Y2NmNDMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMjhUMDc0MzU1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjY1M2YyYjdkMGJhOTg2MGY3NGZjYWVkNzAwYTMxYTNhNmM2MjAzMTdiZjE0ZTNhMWJjZjA2NjZkZTNmOTQzOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.7vMwmxtuAI2bxgb3UCsrzTYBBSrUWcFfwpIAlZEiZLA"
```

Comment:
Is this the new way that images get added to GitHub now? I added them into and issue and tried to grab the URL, unsure if there's a better way to do this :/

Reply:
🙈 looks like it's not working
<img width="383" alt="image" src="https://github.com/primer/design/assets/7265547/29a90f77-b6d9-43f6-9092-4eb7c6de99a6">